user-guide/network/nictag.rst

```diff
@@ -19,10 +23,19 @@ For other use cases, it is possible to add following NIC tags:
 Managing NIC Tags
 =================
 
-The **nictagadm** command is used for working with NIC tags on the compute node. It is possible to add, create or modify NIC tags with it.
+The ``nictagadm`` command is used for working with NIC tags on the compute node. It is possible to add, create or modify NIC tags with it.
+
+.. warning:: NIC tags should only be managed using ``nictagadm``; if you use ``dladm`` to create and/or delete NIC tags, unexpected behavior might occur.
```

Comment:
This does not apply to overlays. There should be link that points from here to overlay creation.

Reply:
Overlays are special case indeed, and I'll document it, but nictagadm collects overlays from overlay_rules.json. If user creates overlay with dladm it will not be shown in nictagadm output and thus not available in the NIC tag dropdown menu